gridt/controllers/leader.py

```diff
@@ -114,21 +98,21 @@ def possible_leaders(
 ) -> Query:
     """Find possible leaders for a user in a movement."""
 
+
     leader_ids = session.query(UserToUserLink.leader_id).filter(
         UserToUserLink.movement_id == movement.id,
-        not_(UserToUserLink.leader_id.is_(None)),
         UserToUserLink.follower_id == user.id,
         UserToUserLink.destroyed.is_(None)
     )
 
-    possible_leaders = [user_to_user_link.follower for user_to_user_link in
-        session.query(UserToUserLink)
+    possible_leaders = [ subscription.user for subscription in
+        session.query(SUB)
```

Comment:
Does SUB mean subscription or subscriber? Probably better to just spell it out completely

Reply:
we set "SUB = Subscription" so I don't think it is necessary to to use SUBSCRIPTION